OgcApi.Net.Features.PostGis/PostGisProvider.cs

```diff
@@ -38,5 +39,24 @@ protected override Geometry ReadGeometry(DbDataReader dataReader, int ordinal, S
 
             return geometryReader.Read((byte[])dataReader.GetValue(ordinal));
         }
+        public override ICollectionSourceOptions DeserializeCollectionSourceOptions(string json, JsonSerializerOptions options)
+        {
+            return JsonSerializer.Deserialize<SqlCollectionSourceOptions>(json, options);
+        }
+
+        public override void SetCollectionOptions(ICollectionsOptions options)
```

Comment:
This method implementation does not depend on any specific field or method. Implementation can be moved into the SqlDataProvider class or even in IDataProvider interface.